tools/harness-thci/OpenThread_BR.py

```diff
@@ -346,16 +346,15 @@ def _beforeRegisterMulticast(self, sAddr='ff04::1234:777a:1', timeout=300):
         self.bash(cmd)
 
     @API
-    def setupHost(self, setDua=False):
+    def setupHost(self, setDp=False, setDua=False):
```

Comment:
I can see why it's fine to use `setDp=False` in non-DUA TCs. 

But are there any tests that require `setDp=False` to pass?

Reply:
Probably not, but it was agreed to have this behavior on the reference host device, i.e., to be able to stop advertising the Domain Prefix on the backbone link.

Reply:
SGTM. Thanks for clarification.